docs/admin/authorization/rbac.md

```diff
@@ -609,12 +611,24 @@ subjects:
   name: user-1
 ```
 
+### Bootstrapping
+
 When bootstrapping the first roles and role bindings, it is necessary for the initial user to grant permissions they do not yet have.
 To bootstrap initial roles and role bindings:
 
 * Use a credential with the `system:masters` group, which is bound to the `cluster-admin` super-user role by the default bindings.
 * If your API server runs with the insecure port enabled (`--insecure-port`), you can also make API calls via that port, which does not enforce authentication or authorization.
 
+### Pod Creation
+
+Users who have ability to create pods in a namespace can potentially escalate their privileges within that namespace.  If a user is granted permission to create pods (or controllers that create pods), the RBAC API does not police the privileges of the pods being created.  This means that users can create pods that access secrets the user herself cannot read, or that run under a service account with different/greater permissions.
```

Comment:
This isn't really specific to pods. For example, if you can create a replication controller, job, deployment, daemonset, replicaset, etc, those can result in the creation of objects that you do not have permission to create directly.

Reply:
True, but I want to specifically point out these examples because they are surprising and have important security consequences.